cmd/cosign/cli/verify_attestation.go

```diff
@@ -49,6 +75,8 @@ func applyVerifyAttestationFlags(cmd *VerifyAttestationCommand, flagset *flag.Fl
 	flagset.BoolVar(&cmd.CheckClaims, "check-claims", true, "whether to check the claims found")
 	flagset.StringVar(&cmd.FulcioURL, "fulcio-url", "https://fulcio.sigstore.dev", "[EXPERIMENTAL] address of sigstore PKI server")
 	flagset.StringVar(&cmd.RekorURL, "rekor-url", "https://rekor.sigstore.dev", "[EXPERIMENTAL] address of rekor STL server")
+	flagset.StringVar(&cmd.PredicateType, "type", "custom", "specify predicate type (default: custom) (slsaprovenance|link|spdx)")
+	flagset.Var(&cmd.Policies, "policy", "specify CUE files will be using for validation")
```

Comment:
i suggest to include in the description the statement "comma separeted list of CUE files".

Reply:
Right now it's not working like that. You should specify all your Cue files with separate `-policy` flags.
For example: 
```shell
cosign verify-attestation -key cosign-keys/cosign.pub -policy=demos/spdx1.cue -policy=demos/spdx2.cue -type spdx devopps/alpine:3.8
```
 Is this command should work as you said?